consensus/consensus_tests/utils_test.go

```diff
@@ -364,17 +364,17 @@ func baseUtilityContextMock(t *testing.T) *modulesMock.MockUtilityContext {
 	ctrl := gomock.NewController(t)
 	utilityContextMock := modulesMock.NewMockUtilityContext(ctrl)
 	persistenceContextMock := modulesMock.NewMockPersistenceRWContext(ctrl)
-	persistenceContextMock.EXPECT().SetProposalBlock(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).AnyTimes()
-	persistenceContextMock.EXPECT().GetBlockHash(gomock.Any()).Return([]byte(""), nil).AnyTimes()
+	persistenceContextMock.EXPECT().GetBlockHash(gomock.Any()).Return("", nil).AnyTimes()
 
 	utilityContextMock.EXPECT().
 		CreateAndApplyProposalBlock(gomock.Any(), maxTxBytes).
-		Return(appHash, make([][]byte, 0), nil).
+		Return(stateHash, make([][]byte, 0), nil).
 		AnyTimes()
 	utilityContextMock.EXPECT().
 		ApplyBlock().
-		Return(appHash, nil).
+		Return(stateHash, nil).
 		AnyTimes()
+	utilityContextMock.EXPECT().SetProposalBlock(gomock.Any(), gomock.Any(), gomock.Any()).Return(nil).AnyTimes()
```

Comment:
@Olshansk This causes tests to fail to compile, since struct `MockUtilityContext` doesn't implement function `SetProposalBlock`.

Reply:
Did this happen to you locally?

Note how the CI checks passed:

<img width="862" alt="Screenshot 2022-12-13 at 4 44 43 PM" src="https://user-images.githubusercontent.com/1892194/207476669-ae5bddd7-708e-423d-8087-30a49f6cbf1f.png">

Here's also a screenshot of it passing on my local machine:

<img width="1608" alt="Screenshot 2022-12-13 at 4 45 11 PM" src="https://user-images.githubusercontent.com/1892194/207476734-5ff5f724-f3d7-4849-86b5-977eb68ac9d4.png">

And in my `utility_module_mock.go`:

<img width="1608" alt="Screenshot 2022-12-13 at 4 46 40 PM" src="https://user-images.githubusercontent.com/1892194/207476899-9527c540-236a-47a3-b93e-97839d87b9a9.png">


When we change the interfaces (and in turns the mocks on which the interfaces are built), we need to run `make mockgen`, so they are recompiled. I like to run `make develop_test` whenever I want to do a clean end-to-end.

Lmk if that makes sense